flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -2092,6 +2262,19 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
     public static final BuiltInFunctionDefinition IS_JSON =
             BuiltInFunctionDefinition.newBuilder()
                     .name("IS_JSON")
+                    .callSyntax(
+                            (sqlName, operands) -> {
+                                final String s =
+                                        String.format(
+                                                "%s IS JSON",
+                                                CallSyntaxUtils.asSerializableOperand(
+                                                        operands.get(0)));
+                                if (operands.size() > 1) {
```

Comment:
can you add more comments for this special cases? It's hard to understand what is going on here.